datadog_checks_dev/datadog_checks/dev/_env.py

```diff
@@ -135,6 +142,16 @@ def replay_check_run(agent_collector, stub_aggregator, stub_agent):
                     data.get('device'),
                 )
 
+        for ep_event_type in EVENT_PLATFORM_EVENT_TYPES:
+            ep_events = aggregator.get(ep_event_type) or []
```

Comment:
How does `aggregator.get` actually load the events? I'm not sure how this works. Shouldn't we be calling `aggregator.get_event_platform_events` ?

Reply:
@djova the "aggregator" here is actually just a python dict, it correspond to the "aggregator" section in the example json below.

The whole `agent_collector` dict is like this:

```
[
  {
    "aggregator": {
      "dbm-activity": null,
      "dbm-metrics": null,
      "dbm-samples": null,
      "metrics": [
        ...
      ],
      "network-devices-metadata": [
        {
          "EventType": "network-devices-metadata",
          "UnmarshalledEvent": {
            ...
        }
      ],
      "service_checks": [
        ...
      ]
    },
    "inventories": {},
    "runner": {
      "AverageExecutionTime": 267,
      ...
    }
  }
]
```
The output is from for example: `agent check snmp --json`

Reply:
Ohhh I see we're parsing the output of the real agent. Cool!